### PR TITLE
For beach, makes shallow->deep edge nicer.

### PR DESCRIPTION
**Pull request by Tastyfish**
Makes the formally blocky seam have sprites for all directions, as well as a cleaner overlay transition, & and maximizes sand lumpiness, adds a little bit of depth.

Preview

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Looks a lot better. Thank you.

**Comment by FalseIncarnate** [comment]
Thanks for this, looks way better. My spriting skills are pretty lackluster, so I didn't try doing this one myself.

**Comment by Tastyfish** [comment]
No problem, it was fun to learn how to do sprite smoothed icons properly.

File: code/game/turfs/unsimulated/beach.dm [hunk 1]
```diff
@@ -34,11 +34,35 @@
...
+	canSmoothWith = list(
+		/turf/unsimulated/beach/water/drop, /turf/unsimulated/beach/water/drop/dense,
+		/turf/unsimulated/beach/water, /turf/unsimulated/beach/water/dense,
+		/turf/unsimulated/beach/water/edge_drop)
 
-/turf/unsimulated/beach/water/dense			//for boundary "walls"
+/turf/unsimulated/beach/water/drop/New()
+	..()
+	smooth_icon(src)
+	spawn(1)
```
Comment (Fox-McCloud) [inline code/game/turfs/unsimulated/beach.dm]: ewww.

**Comment by KasparoVy** [comment]
Edgy, brooding beach

**Pull request merged**